tf2onnx/onnx_opset/tensor.py

```diff
@@ -1464,18 +1465,40 @@ def version_10(cls, ctx, node, **kwargs):
         # onnx output is [num_selected_boxes, 3], the meaning of last dim is [batch_index, class_index, box_index]
         # while tf's output is [num_selected_boxes]
         ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[0], axes=[0])
-        ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[1], axes=[0, 1])
+        input_score = ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[1], axes=[0, 1])
         ctx.insert_new_node_on_input(node, "Cast", node.input[2], to=onnx_pb.TensorProto.INT64)
         # replace original node with nonmaxsurppress + slice + squeeze + cast
-        dtypes = [ctx.get_dtype(node.output[0])]
-        shapes = [ctx.get_shape(node.output[0])]
+        dtypes = [[ctx.get_dtype(output)] for output in node.output]
+        shapes = [[ctx.get_shape(output)] for output in node.output]
+        max_output_size = node.input[2]
         ctx.remove_node(node.name)
-        new_nonmaxsurppress = ctx.make_node(node.type, node.input).output[0]
+        new_nonmaxsurppress = ctx.make_node(node.type, node.input[: 5]).output[0]
         slice_op = GraphBuilder(ctx).make_slice({"data": new_nonmaxsurppress,
                                                  "axes": [1], "ends": [3], "starts": [2]})
         squeeze_op = ctx.make_node("Squeeze", [slice_op], attr={"axes": [1]})
-        ctx.make_node("Cast", inputs=squeeze_op.output, attr={"to": onnx_pb.TensorProto.INT32},
-                      name=node.name, outputs=node.output, dtypes=dtypes, shapes=shapes)
+        if len(node.input) > 5:  # V5
+            logger.warning("NonMaxSuppressionV5 only parltially supported, soft_nms_sigma must be 0.0")
```

Comment:
Is soft_nms_sigma usually 0.0? 

Is there a way to check the value, and then throw the warning? -- it seems like this will throw a warning at every V5 node at conversion time, and if the user ignores the warning, at runtime they'll just get a wrong result. Maybe I'm reading this wrong however.

Reply:
Of course it could by any designated value.
Better to throw an exception during conversion on non-zero soft_nms_sigma, but it is an input meaning could be a non-const.

Reply:
How does the code snippet below look instead? It should throw exception if it cannot handle the ops, otherwise the generated model should be guaranteed to work.

```python
utils.make_sure(node.inputs[index], "node must be const")
val = node.inputs[index].get_tensor_value(False)
utils.make_sure(val == 0, "err msg...")
```

Reply:
Done, :)